MRFIXIT: unify the makefile - no point in having a separate platform for rockpro
--- a/Makefile
+++ b/Makefile
@@ -407,11 +407,11 @@
     CXXFLAGS += -march=armv8-a+crc -mfpu=neon-fp-armv8
 
     ifneq (,$(findstring RK3399,$(platform)))
-      CFLAGS += -mtune=cortex-a72.cortex-a53
-      CXXFLAGS += -mtune=cortex-a72.cortex-a53
+      CFLAGS += -mcpu=cortex-a72 -mtune=cortex-a72.cortex-a53 $(CPUFLAGS)
+      CXXFLAGS += -mcpu=cortex-a72 -mtune=cortex-a72.cortex-a53 $(CPUFLAGS)
     else ifneq (,$(findstring RK3328,$(platform)))
-      CFLAGS += -mtune=cortex-a53
-      CXXFLAGS += -mtune=cortex-a53
+      CFLAGS += -mcpu=cortex-a53 -mtune=cortex-a53 $(CPUFLAGS)
+      CXXFLAGS += -mcpu=cortex-a53 -mtune=cortex-a53 $(CPUFLAGS)
       CORE_DEFINES += -DLOW_END
     endif
 
